ui/src/shared/models/info.ts

```diff
@@ -1,6 +1,7 @@
 export interface Link {
     name: string;
     scope: string;
+    type?: string;
```

Comment:
open for discussion,
I'm thinking if using `target` makes more sense, user can use value as if its HTML target attribute. `_blank`, `_self`
`internal` is not very precise since its opening the link in the same tab

Reply:
I agree with @tczhao